core-bundle/assets/scripts/core.js

```diff
@@ -2112,6 +2112,10 @@ window.Theme =
 	 * Set up the textarea resizing
 	 */
 	setupTextareaResizing: function() {
+		if (window.console) {
+			console.warn('Theme.setupTextareaResizing() is deprecated. Please use the Stimulus controller instead.');
+		}
```

Comment:
Unsure if we need to deprecate them as this is not bound to `Backend` but `Theme`. I did comment out the init due to it being very global (on every `tl_textarea`).

Since it's part of the `Theme.` methods, should we simply remove it or should we leave it here as kind of an artifact in case people do the initialization in their own backend JavaSripts?

Reply:
We previously decided that these deprecations are not necessary, iirc.